frontend/scss/components/atoms/pixi-scale.scss

```diff
@@ -122,6 +122,65 @@
         border: 2px solid color('white');
       }
     }
+
+    &-indicator {
+      position: absolute;
+      left: 0;
+      top: -30px;
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+      width: 0;
+      font-size: 14px;
+      white-space: nowrap;
+      font-weight: bold;
+      color: color('malachite');
+      transform: translateX(7px);
+
+      &:after {
+        content: '';
+        width: 0;
+        height: 0;
+        margin: 0 0 0 6px;
+        border: 7.5px solid transparent;
+        border-bottom: 0;
+        border-top-color: color('malachite');
```

Comment:
Couldn't this use `currentColor` instead? This would also save you the additional `&:after` definitions in 153ff and 160ff